src/game-of-life/time-profiling.md

```diff
@@ -28,8 +28,8 @@ Calling `performance.now` has very little overhead, so we can create simple
 measurements from it without distorting the performance of the rest of the
 system.
 
-For example, we can create a simple FPS counter that we update on each iteration
-of our `renderLoop`:
+For example, we can create a simple FPS (frames per second) counter that we
```

Comment:
Let's flip this to "frames per second (FPS)"

Reply:
Ack